### PR TITLE
ci: fix publish typos

### PR DESCRIPTION
**Pull request by gord5500**
### Why

Couples typos lying around. After being able to run the workflow I found them all and should be sorted

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -24,23 +24,23 @@ on:
...
-          RELEASE_TAG: ${{ github.event.client_payload.release_tag }}
-          PRERELEASE: ${{ github.events.input.prerelease }}
+          RELEASE_TAG: ${{ github.event.inputs.release_tag }}
+          PRERELEASE: ${{ github.event.inputs.prerelease }}
           GH_TOKEN: ${{ github.token }}
         shell: pwsh
         run: |
           if ($env:PRERELEASE -eq 'true') {
-              $releaseUrl = gh release create "$env:RELEASE_TAG" --draft --prerelease --generate-notes --title "Release $env:RELEASE_TAG"
+              $releaseUrl = gh release create "$env:RELEASE_TAG" --repo Brightspace/bmx --draft --prerelease --generate-notes --title "Release $env:RELEASE_TAG"
```
[the remark this answers is not in the document]
Comment (boarnoah) [reply .github/workflows/publish.yml:41]: Docs makes it sound like that is only needed when creating releases on other repos are you sure this is needed? AFAICS the github actions env has the context on which repo its running in without running the checkout action https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

[the remark this answers is not in the document]
Comment (gord5500) [reply .github/workflows/publish.yml:41]: Here's the two failed runs where I had to add that in for the create release step and this one 

Adding repo flag in fixed the issue but do you think there is something else that's wrong?

https://github.com/Brightspace/bmx/actions/runs/5026838067/jobs/9015589752
https://github.com/Brightspace/bmx/actions/runs/5026886560/jobs/9015755692

[the remark this answers is not in the document]
Comment (boarnoah) [reply .github/workflows/publish.yml:41]: I think that has to do with ``gh release`` wanting to create and push a git tag as well as publish something in the release page. 

We can try with ``--repo``, but I suspect we will need to add a checkout step here in the end (lets try with ``--repo`` first)

**Review by boarnoah — APPROVED** [review]
Sorry meant to approve this earlier

**Pull request merged**